Overview.src.html

```diff
@@ -4027,6 +4039,51 @@ <h3 id=http-cors-protocol>CORS protocol</h3>
  <li><p>Return <var title>p</var>.
 </ol>
 
+<h3>Fetch termination initiated by the user agent</h3>
+
+<p>The user agent may <span title=concept-fetch-terminate>terminate</span> an ongoing fetch with
+reason <i title>garbage collection</i> if that termination is not observable through script.
```

Comment:
Perhaps clarify here that if the script communicates with the server through a side-channel that does not count?